bazel/patches/wasmtime-0.20.0-fix-include-path.diff

```diff
@@ -0,0 +1,28 @@
+diff --git a/crates/c-api/include/wasi.h b/crates/c-api/include/wasi.h
+index 92271496c..35a29069f 100644
+--- a/crates/c-api/include/wasi.h
++++ b/crates/c-api/include/wasi.h
+@@ -7,7 +7,7 @@
+ #ifndef WASI_H
+ #define WASI_H
+ 
+-#include "wasm.h"
++#include "wasmtime/include/wasm.h"
```

Comment:
You should be able to add `wasmtime/include` to include path via `includes` in `cc_library`, or does it collide with `include/proxy-wasm/wasm.h`? See: https://github.com/envoyproxy/envoy/blob/master/bazel/external/wee8.BUILD

Reply:
fixed (I remove wasmtime specific dependency since I no longer need wat2_wasm)

9f267ef
e6cea79